### DISCUSSION TITLE
Make callsign textbox search in QSO's for same contest to prevent duplicates

### DISCUSSION THREAD

**Discussion by PA4RB** [magicbug/Cloudlog#1729] [Ideas / Feature Requests]
While using the  LIVE Contest module I found it useful to use the searchbox. But it would be more convenient to have a live search when entering the callsign textbox
![Search](https://user-images.githubusercontent.com/60810525/198833121-c500384a-df1a-435d-b50d-5d3f59204f0d.png)

**Comment by AndreasK79** [comment]
#1730 added.
